aptos-move/framework/aptos-framework/doc/aptos_governance.md

```diff
@@ -2064,79 +2064,6 @@ The same as spec of <code><a href="aptos_governance.md#0x1_aptos_governance_crea
 </code></pre>
 
 
-<code>stake_pool</code> must exist StakePool.
```

Comment:
This was introduced in this PR: https://github.com/aptos-labs/aptos-core/pull/9700 by setting `verify` to false for the toggle_features function.